exchange/bitswap/bitswap.go

```diff
@@ -208,6 +208,12 @@ func (bs *Bitswap) GetBlocks(ctx context.Context, keys []key.Key) (<-chan *block
 		log.Event(ctx, "Bitswap.GetBlockRequest.Start", &k)
 	}
 
+	go func() {
```

Comment:
Where should the timeout be set then, 1. hardcoded in blockservice, or 2. core.IpfsNode ?